stl/src/ppltasks.cpp

```diff
@@ -350,7 +350,7 @@ namespace Concurrency {
 } // namespace Concurrency
 
 #ifdef _CRT_APP
-extern "C" void __cdecl __crtCleanupCausalityStaticFactories() {
+extern "C" void __cdecl __crtCleanupCausalityStaticFactories() noexcept {

```

Comment:
#4087 is removing this function; I'll pre-emptively resolve the conflict by dropping this change.